MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1187 @@
+
+The `DataFormAutoCompleteEditor` provides display the filtered suggestions.
+
+The `AutoCompleteMode` property is used to choose the suggestion display mode in the `DataFormAutoCompleteItem` class.
+
+#### AutoComplete editor suggestion options
+
+The phenomenon of string comparison for filtering suggestions can be changed using the `"StartsWith"` and `"Contains"` property. The default filtering strategy is “StartsWith”, and it is case insensitive. The available filtering modes are,
+
+* StartsWith
+
+* Contains
+
+#### Filtering words that starts with the input text
+
+Displays all the matches that contain first character of the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Country" && e.DataFormItem is DataFormAutoCompleteItem autoCompleteItem)
+        {
+            autoComplete.TextSearchMode = DataFormTextSearchMode.StartsWith;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Filtering words that contains the input text
+
+Displays all the matches that contain the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Country" && e.DataFormItem is DataFormAutoCompleteItem autoCompleteItem)
+        {
+            autoComplete.TextSearchMode = DataFormTextSearchMode.Contains;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Setting the height for autocomplete list item 
+
+By default the autocomplete drop down list item height is `400d` , You can change the size for the drop down list item size by using `MaxDropDownHeight` property in the `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Country" && e.DataFormItem is DataFormAutoCompleteItem autoCompleteItem)
+        {
+            autoComplete.MaxDropDownHeight = 400;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Picker editor
+
+In the picker editor, the [Picker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/picker?view=net-maui-7.0) will be loaded.
+
+`Image`
+
+#### Changing the ItemsSource of Picker
+
+By default, the `ItemsSource` for picker is auto-generated for enum type and collection type properties. For other types, you can set the `ItemsSource` by using `IDataFormSourceProvider`.
```

Comment:
By default, the `ItemsSource` for the picker is auto-generated for enum type property. For other types, you can set the `ItemsSource` by using `IDataFormSourceProvider`.

Reply:
content updated